app.ts

```diff
@@ -34,7 +36,7 @@ app.get('/status', function (req, res) {
 });
 
 app.get('/oparl', (req, res) => {
-  res.redirect('/oparl/System');
+  res.redirect('/oparl/oparl/System');
```

Comment:
what does this do? is this to be consistent with the oparl api somehow? can we get a comment that explains it?

Reply:
Deleted https://github.com/lblod/oparl-to-eli-service/commits/feature/eli-mapping/

Reply:
redirection is not necessary